apps/worker/src/batch_announcer/batch.announcer.ts

```diff
@@ -50,7 +50,7 @@ export class BatchAnnouncer {
     const writer = await ParquetWriter.openStream(parquetSchema, publishStream as any, writerOptions);
     // eslint-disable-next-line no-restricted-syntax
     for await (const announcement of announcements) {
-      await writer.appendRow(announcement);
+      await writer.appendRow(announcement as Record<string, any>);
```

Comment:
No clue why after updates, this error started popping, got fixed by a cast but not sure if its correct

```

> ERROR in ./apps/worker/src/batch_announcer/batch.announcer.ts:53:30
> TS2345: Argument of type 'Announcement' is not assignable to parameter of type 'Record<string, unknown>'.
>   Type '{ announcementType: AnnouncementType; fromId: string; } & TombstoneFields' is not assignable to type 'Record<string, unknown>'.
>     Index signature for type 'string' is missing in type '{ announcementType: AnnouncementType; fromId: string; } & TombstoneFields'.
>     51 |     // eslint-disable-next-line no-restricted-syntax
>     52 |     for await (const announcement of announcements) {
>   > 53 |       await writer.appendRow(announcement);
>        |                              ^^^^^^^^^^^^
>     54 |     }
>     55 |     await writer.close();
>     56 |
> 
> webpack 5.90.1 compiled with 1 error in 13268 ms
> 

```

Reply:
I checked and this is an issue with Parquetjs. I'll work on getting a fix in over there.

Reply:
@saraswatpuneet I found the correct fix and it isn't in the Parquetjs side.

The TypedAnnouncement type needs to explicitly ~extend~ intersect with the `Record<string, unknown>`
<img width="715" alt="image" src="https://github.com/AmplicaLabs/content-publishing-service/assets/1252199/21cc8113-3f94-4192-882e-03c94ee4b3ed">